### PR TITLE
'Algorithm'

### PR DESCRIPTION
**Pull request by Annlix**
一个小小的demo。能力有限，还望指正。

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+var theCompaniesofDrinkingBlood = ["JD"];
```
Comment (Tiotao) [inline src/index.js:1]: use `const` if we are not re-assigning the variable. avoid using `var`

**Review by bofeiw — COMMENTED** [review]
哈哈

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    {
+        if(theCompaniesofDrinkingBlood.includes(this.inaugurationCompany)){
+            let currentDate=new Date();
+            while(this.alive){
+                if(this.bodyStatus!=="sick"){
+                    if(workdays.includes(theWeek[currentDate.getDay()])){
+                        if(currentDate.getHours()>=this.theTimeofWork&&currentDate.getHours()<=this.theTimeofOffWork){
+                            this.workStatus="working";
+                        }else{
+                            if(Math.random()>0.001){
```
Comment (Tiotao) [inline src/index.js]: make 0.001 a const. try not to use magic numbers in the conditionals.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        this.workStatus = "working";// working、sleeping
+        this.inaugurationCompany = inaugurationCompany;
+        this.severityofDisease = null;
+    }
+    work()
+    {
+        if(theCompaniesofDrinkingBlood.includes(this.inaugurationCompany)){
+            let currentDate=new Date();
+            while(this.alive){
+                if(this.bodyStatus!=="sick"){
```
Comment (Tiotao) [inline src/index.js:31]: you can use enums like this 
```
const bodyStatus = {
    SICK: 'sick',
    NORMAL: 'normal',
    ...
}
```

**Review by Tiotao — CHANGES_REQUESTED** [review]
some style issues.

**Pull request closed without merging**